game/state/city/vehicle.cpp

```diff
@@ -42,6 +42,28 @@ namespace OpenApoc
 namespace
 {
 static const float M_2xPI = 2.0f * M_PI;
+float xyToFacing(const Vec2<float> &xy)
```

Comment:
I'd love this to be constexpr, but I guess if we still want c++11 compat it'll be less clear to put it on a single like (this was relaxed for c++14 IIRC)